packages/api/src/lib/job-portal-api.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import { getAuthHeaders } from './lms';
+import { RecruiterData, StudentData } from './job-portal';
+
+export type CreateStudentProfileRequest = StudentData & { userId: string };
+export async function createStudentProfile(data: CreateStudentProfileRequest) {
+  const resp = await axios.post('/api/job-portal/create-student-profile', data, {
+    headers: getAuthHeaders(),
+  });
+  return resp.data;
+}
+
+export type createRecruiterProfileRequest = RecruiterData & { userId: string };
+export async function createRecruiterProfile(data: createRecruiterProfileRequest) {
+  const resp = await axios.post('/api/job-portal/create-recruiter-profile', data, {
+    headers: getAuthHeaders(),
+  });
+  return resp.data;
+}
+
+export async function getStudentProfile() {
+  return axios.get('/api/job-portal/get-student-profile', {
+    headers: getAuthHeaders(),
+  });
+}
+
+export async function getRecruiterProfile() {
+  return axios.get('/api/job-portal/get-recruiter-profile', {
+    headers: getAuthHeaders(),
+  });
```

Comment:
All functions like this (API specs) should have a fixed return type so that the caller doesn't need to guess.

```suggestion
  const resp = await axios.get('/api/job-portal/get-recruiter-profile', {
    headers: getAuthHeaders(),
  });
  return resp.data as RecruiterData;
```

Reply:
got it, made changes.